docs/src/content/docs/fr/guides/i18n.mdx

```diff
@@ -67,6 +67,8 @@ Starlight offre une prise en charge intégrée des sites multilingues, y compris
 
 </Steps>
 
+Pour les scénarios i18n plus avancés, Starlight prend également en charge la configuration de l'internationalisation à l'aide de l'option de [configuration `i18n` dans Astro](https://docs.astro.build/fr/guides/internationalization/#configure-i18n-routing).
```

Comment:
```suggestion
Pour les scénarios i18n plus avancés, Starlight prend également en compte la configuration de l'internationalisation à l'aide de l'option de [configuration `i18n` d'Astro](https://docs.astro.build/fr/guides/internationalization/#configure-i18n-routing).
```

I think ["prendre en compte"](https://fr.wiktionary.org/wiki/prendre_en_compte) is closer to what is happening under the hood in this case compared to ["prendre en charge"](https://fr.wiktionary.org/wiki/prendre_en_charge).